### PR TITLE
Update messages_nl.properties

### PR DESCRIPTION
**Pull request by Remski01**
Changed English lines to Dutch.

### CODE REVIEW

**Comment by Remski01** [comment]
I also took a few other with me, (I changed more then i said on https://github.com/EssentialsX/Essentials/issues/1887 

Its because i saw those too when i was scrolling and editing the english one's.

PS: Its the first time ever i used the PR on any plugin. So sorry if i screwed something up or make an mistake with making the PR.

File: Essentials/src/messages_nl.properties [hunk 1]
```diff
@@ -530,7 +530,7 @@ tradeSignEmptyOwner=\u00a74Er is niets te innen bij dit handelsbord.
 treeFailure=\u00a74Fout bij het genereren van de boom. Probeer het opnieuw op gras of aarde.
 treeSpawned=\u00a76Boom gegenereerd.
 true=\u00a7ajuist\u00a7r
-typeTpacancel=\u00a76To cancel this request, type \u00a7c/tpacancel\u00a76.
+typeTpacancel=\u00a76om dit te annuleren, typ \u00a7c/tpacancel\u00a76.
```
Comment (mdcfe) [inline Essentials/src/messages_nl.properties]: This like should start with a capital letter.

File: Essentials/src/messages_nl.properties [hunk 2]
```diff
@@ -57,25 +57,25 @@ chatTypeLocal=[L]
 chatTypeSpy=[Spion]
 cleaned=Gebruikersbestanden opgeschoont.
 cleaning=Opschonen van gebruikersbestanden.
-clearInventoryConfirmToggleOff=\u00a76You will no longer be prompted to confirm inventory clears.
-clearInventoryConfirmToggleOn=\u00a76You will now be prompted to confirm inventory clears.
-commandCooldown=\u00a7cYou cannot type that command for {0}.
+clearInventoryConfirmToggleOff=\u00a76Je zal niet meer worden gevraagd om de inventory clear te bevestigen.
+clearInventoryConfirmToggleOn=\u00a76\u00a76Je zal nu worden gevraagd om de inventory clear te bevestigen.
```
Comment (mdcfe) [inline Essentials/src/messages_nl.properties]: There is a duplicate `\u00a76` on this line.

**Review by mdcfe — CHANGES_REQUESTED** [review]
Just a couple of minor things that need to be taken care of, but otherwise this looks good.

**Review by mdcfe — APPROVED** [review]

**Pull request merged**